scene/2d/node_2d.cpp

```diff
@@ -408,6 +408,14 @@ Point2 Node2D::to_global(Point2 p_local) const {
 	return get_global_transform().xform(p_local);
 }
 
+Point2 Node2D::to_local_rotation(Point2 p_global) const {
+	return get_global_transform().basis_xform_inv(p_global);
+}
+
+Point2 Node2D::to_global_rotation(Point2 p_local) const {
+	return get_global_transform().basis_xform(p_local);
+}
```

Comment:
This is wrong. The name implies rotation only, but the actual behavior of `basis_xform` also involves scale and shearing.

Reply:
True, this should be named to_local/global_basis as I did for Node3D. I must say I primarily thought of rotations, but scale has an effect as well.